### PR TITLE
Fix galaxy_tool_util for composite file input testing in workflows.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: lib/galaxy/tool_util/verify/interactor.py [hunk 1]
```diff
@@ -146,6 +146,56 @@ def get_tool_tests(self, tool_id, tool_version=None):
         assert response.status_code == 200, "Non 200 response from tool test API. [%s]" % response.content
         return response.json()
 
+    def verify_output_collection(self, output_collection_def, output_collection_id):
+        name = output_collection_def.name
+        data_collection = galaxy_interactor._get("dataset_collections/%s" % output_collection_id, data={"instance_type": "history"}).json()
```
Comment (mvdbeek) [inline lib/galaxy/tool_util/verify/interactor.py]:
```suggestion
        data_collection = self._get("dataset_collections/%s" % output_collection_id, data={"instance_type": "history"}).json()
```

File: lib/galaxy/tool_util/verify/interactor.py [hunk 2]
```diff
@@ -146,6 +146,56 @@ def get_tool_tests(self, tool_id, tool_version=None):
...
+                element = get_element(element_objects, element_identifier)
+                if not element:
+                    template = "Failed to find identifier [%s] for testing, tool generated collection elements [%s]"
+                    message = template % (element_identifier, element_objects)
+                    raise AssertionError(message)
+
+                element_type = element["element_type"]
+                if element_type != "dataset_collection":
+                    hda = element["object"]
+                    galaxy_interactor.verify_output_dataset(
```
Comment (mvdbeek) [inline lib/galaxy/tool_util/verify/interactor.py]:
```suggestion
                    self.verify_output_dataset(
```

**Comment by bgruening** [comment]
@jmchilton is this still WIP?

**Comment by jmchilton** [comment]
Out of WIP - thanks for the ping.

**Pull request merged**